View/LoginScreen/login_screen.kv

```diff
@@ -1,4 +1,80 @@
 <LoginScreenView>:
-    Button:
-        text: "Click Me Bruh!"
-        on_release: root.manager.current = "profile screen"
+    MDFloatLayout:  
+        # md_bg_color: 0, 1, 1, .5
+        size_hint: 1, 0.4
+        pos_hint: {"center_x": .5, "center_y": 0.78}
+        Image: 
+            source: "assets/images/FITNITIE.png"
+            size_hint: 1, 1
+            pos_hint: {"center_x": .5, "center_y": 0.8}
+
+        Image: 
+            source: "assets/images/ICON.png"
+            size_hint: 1, 1
+            pos_hint: {"center_x": .5, "center_y": 0.3}
+    
+    MDFloatLayout:
+        size_hint: 1, 1.055
+        pos_hint: {"center_x": 0.9, "center_y": 0.8}
+        MDLabel:
+            text: "LOG-IN"
+            halign: "center"
+            font_name: "Poppins-Medium"
+            font_size: 40
+
+    MDBoxLayout:  
+        orientation: 'vertical'
+        size_hint: 0.65, 0.4
+        pos_hint: {"center_x": .5, "center_y": .26}
+        # md_bg_color: 0, 1, 1, .5
+        # line_color: 0, 0, 1, 1
+        MDTextField:
```

Comment:
since same lang yung ibang properties ng `MDTextField` ng username at password, pwede mo gawing dynamic class sila para mahide yung details na yon and mas maikli. pero nasayo pa rin naman if ayaw mo